pkg/restore/backoff.go

```diff
@@ -0,0 +1,115 @@
+package restore
+
+import (
+	"time"
+
+	"github.com/pingcap/errors"
+	"github.com/pingcap/log"
+	"go.uber.org/zap"
+
+	"github.com/pingcap/br/pkg/utils"
+)
+
+var (
+	errNotLeader           = errors.New("not leader")
+	errKeyNotInRegion      = errors.New("key not in region")
+	errResp                = errors.New("response error")
+	errRewriteRuleNotFound = errors.New("rewrite rule not found")
+	errRangeIsEmpty        = errors.New("range is empty")
+	errGrpc                = errors.New("gRPC error")
```

Comment:
These will record the stack of `restore.init()`, making the stack trace useless. Unless you replace all calls of `errors.Trace()` with `errors.WithStack()`, please construct these errors using a stack-less constructor such as the [standard `errors.New`](https://pkg.go.dev/errors?tab=doc#New).